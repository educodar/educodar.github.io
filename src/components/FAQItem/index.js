import React from "react"
import AnswerText from "../AnswerText"
import QuestionText from "../QuestionText"
import Box from "@material-ui/core/Box"
import Divider from "@material-ui/core/Divider"

const FAQItem = ({ question, children }) => (
  <Box itemScope itemProp="mainEntity" itemType="https://schema.org/Question">
    <QuestionText>{question}</QuestionText>
    <AnswerText>{children}</AnswerText>
    <Divider />
  </Box>
)

export default FAQItem
