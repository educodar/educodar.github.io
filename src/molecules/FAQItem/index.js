import React from "react"
import QuestionText from "../../atoms/QuestionText"
import AnswerText from "../../atoms/AnswerText"

export default ({ question, children }) => (
  <>
    <div itemScope itemProp="mainEntity" itemType="https://schema.org/Question">
      <QuestionText>{question}</QuestionText>
      <AnswerText>{children}</AnswerText>
    </div>
  </>
)
