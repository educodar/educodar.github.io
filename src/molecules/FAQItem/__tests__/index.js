import React from "react"
import renderer from "react-test-renderer"
import FAQItem from ".."

describe("FAQ Item component", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<FAQItem />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
