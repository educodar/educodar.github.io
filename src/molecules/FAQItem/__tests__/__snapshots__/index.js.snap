// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAQ Item component renders correctly 1`] = `
<div
  itemProp="mainEntity"
  itemScope={true}
  itemType="https://schema.org/Question"
>
  <dt
    itemProp="name"
  />
  <dd
    itemProp="acceptedAnswer"
    itemScope={true}
    itemType="https://schema.org/Answer"
  >
    <div
      itemProp="text"
    />
  </dd>
</div>
`;
